Mail Scripts/Case details/Case details notification.js

```diff
@@ -0,0 +1,110 @@
+(function runMailScript(current, email, email_action, event) {
+    var gr = new GlideRecord('sn_customerservice_case'); // Replace with case table name)
+    var caseLink = '<a href="' + current.getLink() + '" style="font-weight: bold;">' + current.number + '</a>'; // Creates a clickable link to the case record
+    var subject = 'Added to case ' + current.number; // Sets the subject with the current case number
+    email.setSubject(subject);
+
+    // Querying for the case record
+    gr.addQuery('sys_id', current.sys_id);

```

Comment:
`getUniqueValue()` does this but again you don't need this additional GlideRecord at all

Reply:
Removed.